Added more paintings to the same artists

someyaaltous opened the pull request.

SajaLahaleeh committed:
solve conflict between two branch

SajaLahaleeh committed:
solve conflect

SajaLahaleeh committed:
Delete config.env

SajaLahaleeh committed:
remove config.env

SajaLahaleeh committed:
Merge branch 'formEndpoint' of github.com:fack2/Artists8 into formEndpoint

SajaLahaleeh committed:
eslintify files and delete comments

SajaLahaleeh committed:
eslint all files / add eslint script files

SajaLahaleeh committed:
delete hbs comment in form

someyaaltous committed:
Added more paintings to the same artists

jema28 commented on src/controllers/painting.js:
```diff
@@ -1,9 +1,14 @@
 const getPaintingQuery = require('../queries/get4paintings')
+const getotherpaintings = require('../queries/getotherpaintings')
```
Be more consistent with the naming e.g. camel case.

jema28 commented on src/controllers/painting.js:
```diff
@@ -1,9 +1,14 @@
+  getPaintingQuery(req.params.id)
+    .then(result => {
+      getotherpaintings(req.params.id)
+        .then(result2 => {
```
Why result2 here? You can just call it result again.

jema28 replied:
This is still a nested promise. You need to use promise.all().

jema28 commented on src/queries/getotherpaintings.js:
```diff
@@ -1,12 +1,13 @@
-    }
-    cb(null, result.rows)
-  })
+const getPainting2 = id => {
```
Why 2?

SajaLahaleeh committed:
fix assets spelling

SajaLahaleeh committed:
fix spelling in painting form

SajaLahaleeh committed:
add dev script

SajaLahaleeh committed:
refactor router and controller files
- reaarange router index.
- merge get and post painting form
- delete endpoints and eslint
- add spaces in files

jema28 committed:
Merge pull request #12 from fack2/formEndpoint

Form endpoint

someyaaltous committed:
fix merge conflicts

someyaaltous committed:
change ids in sql

someyaaltous committed:
add other paintings in hbs

someyaaltous committed:
fixed conflict

someyaaltous committed:
fixed conflict

someyaaltous committed:
fixed conflict

jema28 commented on package.json:
```diff
@@ -19,7 +19,9 @@
 		"start": "node src/server.js",
-		"dev": "nodemon src/server.js"
+		"dev": "nodemon src/server.js",
+		"eslint":"eslint .",
```
Didn't we have these up already?

jema28 replied:
There should be a colon between eslint and fix and it's a modifier.

jema28 commented on public/style.css:
```diff
@@ -1,15 +1,28 @@
-  border: solid 5px;
+  width: 45%;
+  border-radius: 1px;
+  border: solid 1px;
```
This needs a color otherwise it won't show up.

someyaaltous committed:
Turned nested promimes into promise.all

someyaaltous committed:
fixed eslint
